### ISSUE TITLE
videos fail to play on Radio zamaneh

### ISSUE THREAD

**Issue by Popolechien** [openzim/zim-requests#1104]
I could not find a ticket discussing this already and I was under the impression that this particular zim was done, so here it is:
https://dev.library.kiwix.org/viewer#radiozamaneh-com_far_all_2024-06/www.radiozamaneh.com/819674/

Video is not playing on Chrome.

**Comment by benoit74** [comment]
Indeed, you've already reviewed the ZIM in https://github.com/openzim/zim-requests/issues/830 as far as I've understood.

Anyway, the video is really not working on Chrome with kiwix-serve. It works well on Firefox and Safari with kiwix-serve. 

I did not found any specific error which could lead to an explanation, and since videos are known to be hard to make work, videos are not a corner-stone of radiozamaneh.com, and some browsers seems to be OK, I'm not sure we will be able to solve this issue. I'm tranferring the issue to zimit anyway.
